sdk/core/azure_core_amqp/Cargo.toml

```diff
@@ -18,7 +18,7 @@ categories = ["api-bindings"]
 edition = "2021"
 
 [dependencies]
-async-std.workspace = true
+tokio.workspace = true
```

Comment:
@LarryOsterman we talked about this before, but messaging is going to have a take an async runtime dependency, right? `async-std` already was, assuming any of the executor was used. If we can stick to executor-free abstractions from, say, `futures` great, but I seem to recall we probably can't here. I suppose make sure it's used sparingly enough that it shouldn't be too hard to abstract away later if required.